ibu-imager/ops/ops.go

```diff
@@ -209,3 +217,87 @@ func (o *ops) RunRecert(recertContainerImage, authFile, recertConfigFile string,
 
 	return nil
 }
+
+func (o *ops) ExtractTarWithSELinux(srcPath, destPath string) error {
+	_, err := o.hostCommandsExecutor.Execute(
+		"tar", "xzf", srcPath, "-C", destPath, "--selinux",
+	)
+	return err
+}
+
+func (o *ops) RemountSysroot() error {
+	_, err := o.hostCommandsExecutor.Execute("mount", "/sysroot", "-o", "remount,rw")
+	return err
+}
+
+func (o *ops) ImageExists(img string) (bool, error) {
+	_, err := o.hostCommandsExecutor.Execute("podman", "image", "exists", img)
+	if err != nil {
+		var exitError *exec.ExitError
+		if errors.As(err, &exitError) {
+			if exitError.ExitCode() == 1 {
+				return false, nil
+			}
+			return false, err
+		} else {
+			return false, err
+		}
+	}
+	return true, nil
+}
+
+type PodmanImage struct {
+	Repositories []string `json:"Repositories"`
+}
+
+// IsImageMounted checkes whether certain image is mounted
+// pass in the full address with tag e.g: quay.io/openshift/lifecycle-agent-operator:latest
+func (o *ops) IsImageMounted(imgName string) (bool, error) {
+	output, err := o.hostCommandsExecutor.Execute("podman", "image", "mount", "--format", "json")
+	if err != nil {
+		return false, err
+	}
+	var images []PodmanImage
+	if err := json.Unmarshal([]byte(output), &images); err != nil {
+		return false, err
+	}
+	for _, img := range images {
+		if len(img.Repositories) == 0 {
```

Comment:
nit: With the for loop, this check would now be unnecessary

Reply:
i just saw remember that